app/main.py

```diff
@@ -14,4 +14,8 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    counter = 0
+    for char in phrase:
```

Comment:
The solution is ok, but can you try to rewrite it using just 1 line?

Reply:
Please don't hide comments so that we can see them when reviewing your work next time